ccd-definition/CaseTypeTab.json

```diff
@@ -14,18 +14,28 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "Channel": "CaseWorker",
     "TabID": "HearingTab",
-    "TabLabel": "Hearing",
-    "TabDisplayOrder": 2,
+    "TabLabel": "Hearings",
+    "TabDisplayOrder": 1,
     "CaseFieldID": "hearing",
     "TabFieldDisplayOrder": 1
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "Channel": "CaseWorker",
+    "TabID": "HearingTab",
+    "TabLabel": "Hearings",
```

Comment:
Isn't this tab duplicated now?

Reply:
no there are two types of hearing, the tab itself is not duplicated there are two types of hearings that can be displayed under this tab header @akomisarek

Reply:
Looks ok now.